### ISSUE TITLE
Netrw :MF doesn't select glob patterns (*.c)

### ISSUE THREAD

**Issue by habamax** [vim/vim#5239]
**Describe the bug**
:MF command in netrw doesn't select files with glob patterns

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Run `vim` 
2. Open netrw for directory with C files
3. Type `:MF *.c`
4. Nothing is selected

**Expected behavior**
All `c` files are selected

**Environment (please complete the following information):**
 - Vim version [8.1.2318]
 - OS: [Windows 10]
 - Terminal: GUI

**Additional context**
https://vi.stackexchange.com/questions/18357/netrw-multiple-files-selection/21951#21951

The issue is in `s:NetrwMarkFiles` function where `fnameescape` function is used. It escapes `*` and `glob`  is unable to find a file `\*.c`:

https://github.com/vim/vim/blob/5f763348186007e91932f5134da8dbd0d3bffb11/runtime/autoload/netrw.vim#L6667

https://github.com/vim/vim/blob/5f763348186007e91932f5134da8dbd0d3bffb11/runtime/autoload/netrw.vim#L6669

I have tried it removing `fnamescape` -- it worked just fine.

**Comment by Shane-XB-Qian** [comment]
looks it had been fixed at version#167
so probably this ticket can be closed.
```vim
@@ -6661,15 +6688,15 @@ fun! s:NetrwMarkFiles(islocal,...)
 "  call Dfunc("s:NetrwMarkFiles(islocal=".a:islocal."...) a:0=".a:0)
   let curdir = s:NetrwGetCurdir(a:islocal)
   let i      = 1
   while i <= a:0
    if a:islocal
     if v:version > 704 || (v:version == 704 && has("patch656"))
-     let mffiles= glob(fnameescape(a:{i}),0,1,1)
+     let mffiles= glob(a:{i},0,1,1)
     else
-     let mffiles= glob(fnameescape(a:{i}),0,1)
+     let mffiles= glob(a:{i},0,1)
     endif
    else
     let mffiles= [a:{i}]
    endif
 "   call Decho("mffiles".string(mffiles),'~'.expand("<slnum>"))
    for mffile in mffiles
```

**Comment by k-takata** [comment]
Closing.
